src/main/java/christmas/model/Orders.java

```diff
@@ -0,0 +1,40 @@
+package christmas.model;
+
+import static christmas.config.CommonConfig.ORDERS_REGEX;
+import static christmas.config.CommonConfig.ORDER_MENU_COUNT_REGEX;
+
+import christmas.dto.OrderDto;
+import christmas.util.Validation;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Orders {
+    private final OrderList orders;
+
+    public Orders(String orders) {
+        Validation.validOrders(orders);
+        this.orders = new OrderList(makeList(orders));
+    }
+
+    public List<OrderDto> toDtoList() {
+        return orders.getOrders().stream().map(Order::toDto).collect(Collectors.toList());
```

Comment:
스트림 체인을 사용할 경우 각 단계를 개행으로 구분해 주면 훨씬 가독성이 좋아집니다!
추가적으로 JDK 17부터는 `toList()` 메서드가 제공됩니다!
```suggestion
        return orders.getOrders().stream()
                .map(Order::toDto)
                .toList();
```